scripts/ci-capd-e2e.sh

```diff
@@ -24,5 +24,17 @@ cd "${REPO_ROOT}" || exit 1
 # shellcheck source=./hack/ensure-go.sh
 source "${REPO_ROOT}/hack/ensure-go.sh"
 
+KIND_VERSION=v0.6.1
+GOOS=$(go env GOOS)
+GOARCH=$(go env GOARCH)
+
+install_kind() {
+   wget "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-${GOOS}-${GOARCH}" \
+     --no-verbose -O /usr/local/bin/kind
+   chmod +x /usr/local/bin/kind
+}
+
+install_kind
```

Comment:
Can we use https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/hack/ensure-kind.sh?

Reply:
oh sure, I didn't realize that was there! one sec